VagrantProvisionUbuntu1604.sh

```diff
@@ -96,29 +97,31 @@ sudo apt-get -q -y install \
  libboost-dev \
  libcppunit-dev \
  libcv-dev \
- libgdal-dev \
  libgeos++-dev \
  libglpk-dev \
  libicu-dev \
  liblog4cxx10-dev \
  libnewmat10-dev \
  libogdi3.2-dev \
  libopencv-dev \
+ libpq-dev \
  libproj-dev \
  libprotobuf-dev \
  libqt4-dev \
  libqt4-sql-psql \
  libqt4-sql-sqlite \
  libstxxl-dev \
+ libv8-dev \
  maven \
- npm \
+ node-gyp \
  openssh-server \
  patch \
  pgadmin3 \
  postgresql-9.5 \
  postgresql-9.5-postgis-2.3 \
  postgresql-9.5-postgis-scripts \
  postgresql-client-9.5 \
+ postgis \
```

Comment:
This should be brought in automatically by the PGDG `postgresql-9.5-postgis-2.3` package.  I believe this is the system package which would conflict with PGDG.

Reply:
removed.